### ISSUE TITLE
Systemd fails to compile: missing libacl

### ISSUE THREAD

**Issue by andrewyatz** [Linuxbrew/homebrew-core#4587]
# Please fill out the issue checklist below, and provide all of the requested information.

# Please always follow these steps:
- [x] Confirmed this is a problem with `brew install`ing one, specific formula and not every time you run `brew`? If it's a general `brew` problem please file this issue at https://github.com/Linuxbrew/brew/issues/new.
- [ ] If you have a macOS system and can replicate the problem on macOS with Homebrew, please report the issue to [Homebrew/core](https://github.com/Homebrew/homebrew-core) and follow their issue template from your macOS system. If you are unsure, please report the issue to Linuxbrew.
- [x] Ran `brew update` and retried your prior step?
- [x] Ran `brew doctor`, fixed all issues and retried your prior step?
- [x] Ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?

To help us debug your issue please explain:
- What you were trying to do (and why)

Originally I was trying to install rabbitmq, which depends on erlang. That in turn depends on wxmac and eventually `xorg/mesa` and finally `systemd`. The systemd compile fails because there is no `libacl` available. If I install erlang and skip the wxmac install the code works fine.

- What happened (include command output)

Gist available from: https://gist.github.com/22109d46074baa71e9b2fada87a546a0

- What you expected to happen

Systemd would be installed

- Step-by-step reproduction instructions (by running `brew install` commands)

Pretty much just run `brew install systemd`.

**Comment by iMichka** [comment]
Hi. Thanks for reporting this.

Copied the error here:
```
checking acl/libacl.h usability... yes
checking acl/libacl.h presence... yes
checking for acl/libacl.h... yes
checking for acl_get_file in -lacl... no
configure: error: *** libacl not found.
```

We have 2 solutions here:
- create an `acl` formula and use it
- Try to use the `--disable-acl` flag while building `systemd`.

Could you try to modify the `systemd` formula and add the `--disable-acl` flag, and see if it helps?

**Comment by iMichka** [comment]
I'll disable `acl` for the moment, until somebody works on an `acl` formula. Pr open here: #4802

**Comment by iMichka** [comment]
Fixed by #4802
